lib/recode/task/remove_parens.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Recode.Task.RemoveParens do
+  @shortdoc "Removes parens from locals without parens"
+
+  @moduledoc """
+  Don't use parens for functions that don't need them.
+
+          # preferred
+          assert true == true
+
+          # not preferred
+          assert(true == true)
+
+  This task rewrites the code when `mix recode` runs with `autocorrect: true`.
+  """
+
+  use Recode.Task, corrector: true, category: :readability
+
+  alias Recode.Issue
+  alias Recode.Task.RemoveParens
+  alias Rewrite.Source
+  alias Sourceror.Zipper
+
+  @impl Recode.Task
+  def run(source, opts) do
+    formatter_opts = Mix.Tasks.Format.formatter_opts_for_file(".mix.exs")
```

Comment:
We should fetch the opts with the source path and a fallback.
```suggestion
    formatter_opts = Mix.Tasks.Format.formatter_opts_for_file(source.path || "nofile.ex")
```